pom.xml

```diff
@@ -15,8 +15,10 @@
     <url>http://wiki.jenkins-ci.org/display/JENKINS/Groovy+plugin</url>
 
     <properties>
-        <jenkins.version>1.580.3</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>2.7.3</jenkins.version>
+        <java.level>7</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError> <!-- TODO 12 to fix -->
```

Comment:
🐛 We shouldn't be disabling findbugs to make a PR pass.

Reply:
@jglick That line is new -- if we want to be specific, it prevents findbugs from failing the maven build when an error occurs.   The reason this is a problem is that people can now introduce new issues that Findbugs would otherwise catch.

If you want to FBIgnore the 12 failures until a cleanup PR that's fine -- as noted, it's not in-scope to fix the pre-existing issues though nobody would object if you did -- but globally setting this is risky.